UnitTests/GitUITests/CommandsDialogs/CommitDialog/FormCommitTests.cs

```diff
@@ -10,39 +12,23 @@ namespace GitUITests.CommandsDialogs.CommitDialog
     [Apartment(ApartmentState.STA)]
     public class FormCommitTests
     {
-        // Created once for the fixture
-        private ReferenceRepository _referenceRepository;
-
-        // Created once for each test
+        private ITestRepository _referenceRepository;
         private GitUICommands _commands;
 
         [SetUp]
         public void SetUp()
         {
-            if (_referenceRepository == null)
-            {
-                _referenceRepository = new ReferenceRepository();
-            }
-            else
-            {
-                _referenceRepository.Reset();
```

Comment:
❗️ This behavior is the basis for ensuring performance does not degrade as the integration test suite grows, but was lost in this commit.

Reply:
I think this performance problem does not (and should not) exist. There should not be a single `the integration test suite` for GitExtensions that grows indefinitely, but a growing set of small, independent test repositories relevant to the current test case. Having a continuously growing and changing test suite would slow down maintenance, and make it impossible in the long term.

Please reconsider your opinion.

Reply:
I believe you may have misinterpreted my statement. The reference repository is intended to live for the duration of a test fixture, not for the entire test suite. Some test fixtures may end up with a large number of individual tests, and should not be penalized for doing so.